src/poisson1d/main.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (C) 2017-2018  Waterloo Quantitative Consulting Group, Inc.
+// See COPYING and LICENSE files at project root for more details.
+
+/**
+ *  @file poisson1d/main.cpp ...
+ */
+
+#include "Nodes1DProvisioner.hpp"
+#include "CsvOutputter.hpp"
+#include "Types.hpp"
+#include "Warning.hpp"
+#include "Poisson1d.hpp"
+#include "LSERK4.hpp"
+#include "LinAlgHelpers.hpp"
+#include <blitz/array.h>
+#include <cmath>
+#include <string>
+#include <stdexcept>
+
+using blitz::ColumnMajorArray;
```

Comment:
Switch this with ColumnMajorMatrix in DenseMatrixHelpers.

Reply:
good catch.